toolchain/check/testdata/struct/fail_nested_incomplete.carbon

```diff
@@ -14,9 +14,6 @@ class Incomplete;
 // CHECK:STDERR: ^~~~~~~~~~~~~~~~~
 var s: {.a: Incomplete};
 
-// CHECK:STDERR: fail_nested_incomplete.carbon:[[@LINE+3]]:1: ERROR: Cannot implicitly convert from `<error>*` to `Incomplete*`.
-// CHECK:STDERR: var p: Incomplete* = &s.a;
-// CHECK:STDERR: ^~~~~~~~~~~~~~~~~~~~~~~~~~
 var p: Incomplete* = &s.a;
```

Comment:
Should this be moved out of this fail test, since it no longer fails?

Reply:
This is testing error recovery, and so has to be in a `fail_` file.